pvnet/app.py

```diff
@@ -239,6 +239,12 @@ def app(
     fs = fsspec.open(os.environ["SATELLITE_ZARR_PATH"]).fs
     fs.get(os.environ["SATELLITE_ZARR_PATH"], "latest.zarr.zip")
 
+    # Also download 15-minute satellite if it exists
```

Comment:
ah, I think you dont need to do this, because we have covered this in `ocf_datapipes` 
What do you think?

Reply:
Ah I think I miss read this, 

This is a good solution! One quick question, what happens if that 15 mintue file isnt there?